src/pretalx/submission/models/question.py

```diff
@@ -97,6 +119,32 @@ class Question(LogMixin, models.Model):
             "Do you require an answer from every speaker or for every session?"
         ),
     )
+
+    deadline = models.DateTimeField(
+        null=True,
+        blank=True,
+        verbose_name=_("deadline"),
+        help_text=_(
+            "This field is required for 'require after' and 'freeze after' options "
```

Comment:
More user friendly suggestion (users don't care what a field is etc): "Set a deadline to make this question required or to freeze changes after the given date."

Reply:
I replased the message. 🎀